### PR TITLE
Add 1px Separator in SplitButton

### LINKED ISSUE #986: Update SplitButton
## Summary
Update the SplitButton to a new visual design, separating the button and drop down.

## Rationale
1) Visually separate the two hit targets since they currently blend together and unintentionally look like one hit target
2) Is not visually distinct from the dropdown control (i.e. a split button and a drop down can appear to look like the same control)

image

### PR DESCRIPTION
**Pull request by savatia**
Update the SplitButton to a new visual design, separating the button and drop down.
Visual Comp
Fix #986

### CODE REVIEW

File: dev/SplitButton/SplitButton.xaml [hunk 1]
```diff
@@ -222,21 +218,24 @@
...
+                        <Grid x:Name="PrimaryBackgroundGrid"
+                            Background="{TemplateBinding Background}"/>
 
                         <Grid x:Name="SecondaryBackgroundGrid"
-                            Grid.Column="1"/>
+                            Background="{TemplateBinding Background}"
+                            Grid.Column="2"/>
 
                         <Grid x:Name="Border"
                             Grid.ColumnSpan="2"
```
Comment (teaP) [inline dev/SplitButton/SplitButton.xaml]: Shouldn't this be 3 now?

Comment (savatia) [reply dev/SplitButton/SplitButton.xaml]: Why 3?

Comment (teaP) [reply dev/SplitButton/SplitButton.xaml]: There are now 3 columns, so if there were a border added, the border should go around the entire control. If you set a BorderBrush/BorderThickness on SplitButton now, what does it look like?

Comment (savatia) [reply dev/SplitButton/SplitButton.xaml]: image
It only wraps around the first button. I'll fix it.

Comment (savatia) [reply dev/SplitButton/SplitButton.xaml]: Fixed here fe1c73c

**Pull request merged**